ddtrace/settings/database_monitoring.py

```diff
@@ -14,4 +31,23 @@ class DatabaseMonitoringConfig(En):
     )
 
 
-config = DatabaseMonitoringConfig()
+dbm_config = DatabaseMonitoringConfig()
+
+
+def get_dbm_comment(db_span):
```

Comment:
Not sure if this function should live here or be move to a util module. Leaving here simplifies imports and is easier to read.

Reply:
Leaving it here for now seems fine to me.